ps_emailsmanager.php

```diff
@@ -283,6 +307,11 @@ public function saveTemplateConf()
             //Fetch translations from addons api
             $translations = $this->getEmailsTranslations($language['iso_code']);
 
+            $lang_fields_value = $this->getLangFieldsTranslations($lang_fields, $language['id_lang']);
```

Comment:
Do you really need to create a method for this? Can't you do this in the foreach, where you assign $lang_fields in the first place?